File: Iniciante/1051.py
S = float(input())

if(S <= 2000.00):
    print('Isento')
elif(S >= 2000.01 and S <= 3000.00):
    IR = (S - 2000.00) * 0.08
    print(f'R$ {IR:.2f}')
elif(S >= 3000.01 and S <= 4500.00):
    IR = ((S - 3000.00) * 0.18) + 80.00
    print(f'R$ {IR:.2f}')
elif(S > 4500.00):
    IR = ((S - 4500.00) * 0.28) + 350.00
    print(f'R$ {IR:.2f}')

File: Iniciante/1051.php
<?php

$s = readline();

if($s <= 2000.00){
    echo "Isento\n";
}elseif($s >= 2000.01 && $s <= 3000.00){
    $ir = ($s - 2000.00) * 0.08;
    echo "R$ ".number_format($ir, 2, '.', '')."\n";
}elseif($s >= 3000.01 && $s <= 4500.00){
    $ir = (($s - 3000.00) * 0.18) + 80.00;
    echo "R$ ".number_format($ir, 2, '.', '')."\n";
}elseif($s > 4500.00){
    $ir = (($s - 4500.00) * 0.28) + 350.00;
    echo "R$ ".number_format($ir, 2, '.', '')."\n";
}

?>